sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -130,11 +132,21 @@ internal class ViewAnnotationManagerImpl(
     val viewAnnotation = ViewAnnotation(
       view = inflatedView,
       handleVisibilityAutomatically = options.visible == null,
+      wasVisible = inflatedView.visibility == View.VISIBLE,
```

Comment:
IMO just `isVisible` would work better, it becomes `was` only in future in onGlobalLayout, I suppose

Reply:
done